salt/utils/thin.py

```diff
@@ -21,50 +24,49 @@
 import salt.ext.six as _six
 import tornado
 
+try:
+    import zlib
+except ImportError:
+    zlib = None
+
 # pylint: disable=import-error,no-name-in-module
 try:
     import certifi
-    HAS_CERTIFI = True
 except ImportError:
-    HAS_CERTIFI = False
+    certifi = None
```

Comment:
Isn't SaltStack conventionally using uppercase variable names for those import flags? Or does this only apply to modules?

Reply:
No, not really. We have uppercase vars old style, but we also now have this way too elsewhere. Besides, I'd like to see how then your code will look like when you want to filter-out unavailable modules in `get_tops` but using uppercase vars instead. :wink:

Reply:
This has been an informal convention but we do need to add this sort of thing to a style guide.